### PR TITLE
Data file added, and needed corrections made

### PR DESCRIPTION
**Pull request by nas-cam**
Data needed for testing added in a file. Hardcoded strings removed and implemented different logic for data injection into the test.

### CODE REVIEW

File: internship-automation/pageobjects/myAccountPage.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ class MyAccountPage extends Page{
     }
 
     async enterPassword(password){
-        await this.inputUsername.click();
-        await this.inputUsername.setValue(password);
+        await this.inputPassword.click();
+        await this.inputPassword.setValue(password);
```
Comment (ElvedinS96) [inline internship-automation/pageobjects/myAccountPage.js:28]: Function name says enterPassword, but here you are clicking also. It probably means there should be another clickOnPasswordInput() :)

File: internship-automation/test-suites/smokeTest/smoke-test.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
 describe('001: Smoke test for Shop Tools App', () => {
     it('TC-ID-1. User is able to login, to add item to the cart, and to complete order', async () => {
         await myAccountPage.open();
-        await myAccountPage.enterUsername('username_1');
-        await myAccountPage.enterPassword('strongPssword1?');
+        await myAccountPage.enterUsername(loginData.validUsername);
+        await myAccountPage.enterPassword(loginData.validPassword);
         await myAccountPage.clickOnSubmitButton();
         await expect(securePage.flashAlert).toHaveTextContaining("MY ACCOUNT");
```
Comment (ElvedinS96) [inline internship-automation/test-suites/smokeTest/smoke-test.js]: "MY ACCOUNT" is data also, and all others where you check toHaveTextContaining().

**Review by ElvedinS96 — CHANGES_REQUESTED** [review]

File: internship-automation/pageobjects/myAccountPage.js (hunk 1 above)
Comment (nas-cam) [reply internship-automation/pageobjects/myAccountPage.js:28]: Yes, I forgot about SRP, I will correct this.

File: internship-automation/pageobjects/productPage.js [hunk 3]
```diff
@@ -5,6 +5,9 @@ class ProductPage extends Page{
     get sizeDropdown() { return $('#pa_size'); }
     get buttonAddToCart() { return $('.single_add_to_cart_button.button.alt.wp-element-button'); }
     get buttonViewCart() { return $('a.button.wc-forward.wp-element-button'); }
+    get productPageHeader() { return $('.page-title');}
+    get product() {return $('.noo-product-inner.noo-product-inner2');}
+
```
Comment (ElvedinS96) [inline internship-automation/pageobjects/productPage.js:10]: Extra space?

File: internship-automation/data/commonData.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    lastName : 'Kuhic',
+    company: 'Company',
+    country:'Iceland',
+    address:'642 Vandervort Hill',
+    postCode: '87282-7667',
+    town: 'New Josianne',
+    phone:'670-330-3748',
+    email: 'votysip@abyssmail.com',
+    thankYouNote:'Thank you. Your order has been received.'
+};
```
Comment (ElvedinS96) [inline internship-automation/data/commonData.js]: Missing space at the end?

File: internship-automation/data/commonData.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+export const loginData = {
+    validUsername: 'username_1',
+    validPassword: 'strongPssword1?'
+};
+export const myAccountPageData={
+    searchFieldText:'Search',
+    myAccountHeaderText:'MY ACCOUNT'
+
+};
+
```
Comment (ElvedinS96) [inline internship-automation/data/commonData.js:10]: Can you add spaces in between these sections, like here?

File: internship-automation/test-suites/smokeTest/smoke-test.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        await checkoutPage.enterPhone(checkoutPageData.phone);
+        await checkoutPage.clickOnEmail();
+        await checkoutPage.enterEmail(checkoutPageData.email);
+        await checkoutPage.clickOnTerms();
+        await checkoutPage.clickOnPlaceOrderButton();
+        await expect(checkoutPage.thankYouNote).toHaveTextContaining(checkoutPageData.thankYouNote);
+    });
+});
+
+
```
Comment (ElvedinS96) [inline internship-automation/test-suites/smokeTest/smoke-test.js]: Extra space?

**Review by ElvedinS96 — CHANGES_REQUESTED** [review]

**Review by ElvedinS96 — APPROVED** [review]
This is some great work, well done 👏

**Pull request merged**